src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -87,6 +132,10 @@ void draw_settings_dialog(HostState &host) {
         ImGui::Combo("Log Level", &host.cfg.log_level, "Trace\0Debug\0Info\0Warning\0Error\0Critical\0Off\0");
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Select your preferred log level.");
+        if (ImGui::Button("Apply change")) {
```

Comment:
No need for this, just apply every time a module is added/deleted.

Reply:
no, here is for change log level without reboot no relate lle module